### PR TITLE
🥗 `Gizmo`: System Spec to Add and Remove Gizmo

### PR DESCRIPTION
**Pull request by zspencer**
- https://github.com/zinc-collective/convene/issues/1565
- https://github.com/zinc-collective/convene/issues/1472

For some reason the automated check doesn't *FAIL* despite it failing when we manually execute it!

The "cause" is that the gizmo form has two `_method` fields, one is the hidden `_method` field that is automatically inserted by `form_with` when the `model` is `persisted?`; the other is added by a `button` that sets the `_method` to delete so that it can be removed.

In the system spec; the driven `firefox` takes the `_method` set by the clicked `button`. In the user-land; it sends *both* methods; and Rails picks the first one.

The "solution" will be to ditch the second `_method` (possibly even removing the ability to Remove a Gizmo from the `update` form.)

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -48,6 +48,13 @@ else
   cp .env.example .env
 fi
 
+if test -f ".env.development"; then
+  echo "Found .env.development, leaving it in place!"
+else
+  echo "Copying .env.development.example to .env.development"
+  cp .env.development.example .env
+fi
+
```
Comment (daltonrpruitt) [inline bin/setup:57]: I have a feeling the `.devcontainer/startup.bash` script may need to be updated with this new `.env` changes?
Specifically, the block starting at [line 92](https://github.com/zinc-collective/convene/blob/51ff6f302ac669370e4636e8f3a6baa22a9c13fe/.devcontainer/startup.bash#L92C3-L92) may need to be adjusted.
This assumption may be wrong, but I wanted to bring it up sooner rather than later.

Comment (zspencer) [reply bin/setup:57]: It would be nice if the three places we use (bin/setup, bin/setup-rails, and .devcontainer/startup.bash) all used the same shell script to create these default ffiles; perhaps `bin/setup-env`? Are you available to do that refactor?

File: .env.example [hunk 2]
```diff
@@ -10,11 +10,11 @@
...
-SMTP_PORT=1025
-SMTP_DOMAIN=localhost
-SMTP_AUTHENTICATION=plain
-SMTP_ADDRESS=localhost
-SMTP_ENABLE_TLS=false
+# SMTP_PORT=1025
+# SMTP_DOMAIN=localhost
+# SMTP_AUTHENTICATION=plain
+# SMTP_ADDRESS=localhost
+# SMTP_ENABLE_TLS=false
```
Comment (KellyAH) [inline .env.example:17]: Please break out system test step to it's own PR.

File: bin/setup (hunk 1 above)
Comment (daltonrpruitt) [reply bin/setup:57]: Sure, I can do that.

One script to do a conditionally copy of both `.env.example` and `.env.development.example`?
And then in the `.devcontainer/startup.bash` script, do the `sed` stuff after running this script?
Mostly just probing to make sure I'm understanding what needs to be here.

Comment (zspencer) [reply bin/setup]: Yep! I think the sed stuff should stay in the startup.bash

File: .env.example (hunk 2 above)
Comment (zspencer) [reply .env.example]: Done! Plz review!

**Review by KellyAH — APPROVED** [review]
LGTM
